### ISSUE TITLE
Button focus indicator is invisible on black background

### ISSUE THREAD

**Issue by obulat** [cc-archive/vocabulary-legacy#718]
## Description

The Vocabulary buttons have a box-shadow when they are focused. This looks consistent across browsers, and looks nice on the white background. However, on the black background, for example, in the global header or in the footer, the box-shadow becomes invisible. It is important for accessibility purposes for header/footer, as well as when we have a dark theme for Vocabulary.

We need the designs for additional focus styles to the interactive elements on the dark background. Then we can implement them in Vocabulary.

## Screenshots

The button focus style is visible on the white background as the pink shadow:
![FocusStyle](https://user-images.githubusercontent.com/15233243/95705415-8c948400-0c5c-11eb-9751-0bb4577e0509.png)

The pink shadow is indiscernible on the black background:
![FocusStyleBlack](https://user-images.githubusercontent.com/15233243/95705483-c796b780-0c5c-11eb-92e6-65702c9c065b.png)

## Resolution

- [ ] I would be interested in resolving this bug.

**Comment by fcoveram** [comment]
I would suggest using a white solid border for black background. It is the highest contrast, and I don't recall using white for links. Therefore, white will fit well with all remained colors.

Thanks for bringing this.

**Comment by SaravgiYash** [comment]
@obulat I would like to work on this issue.

**Comment by SaravgiYash** [comment]
@panchovm can you please guide me on what all changes do I need to make and in which files.

**Comment by fcoveram** [comment]
Hi @Yashs911 I will ping @obulat and @dhruvkb, they're in charge of addressing the changes.

**Comment by obulat** [comment]
Go ahead, @Yashs911 !

Here is the line of css for footer buttons : 
https://github.com/creativecommons/vocabulary/blob/bb9a1306b176283a253f4a9b9b0d35ec1170ee05/packages/vocabulary/src/styles/footer.scss#L74

You can add :focus style to it with white outline/border below the :hover style.

**Comment by obulat** [comment]
This issue is closed by #758
